build-gluster-org/scripts/clang-format.py

```diff
@@ -18,10 +18,15 @@
 # Look for any changes applied by clang-format
 changed = subprocess.check_output(['git', 'diff'])
 
+#comment when there is any diff generated after running clang-format
 if changed:
+    with open('comment.txt', 'w') as file:
+    	file.write('CLANG-FORMAT FAILURE:\nBefore merging the patch, this diff needs to be considered for passing clang-format\n\n' + changed.decode("utf-8"))
     print(changed)
```

Comment:
```suggestion
    print("```")
    print(changed)
    print("```")
```

Reply:
You want to format like this for comment text, right?